internal/web_server/web/handlers/quiz_pages_handlers.go

```diff
@@ -31,6 +33,9 @@ func (qph *QuizPagesHandler) RegisterQuizHandlers(e *echo.Group) {
 	e.GET("/fullforte", qph.getFinishedQuizzes)
 	e.GET("/arkiv", qph.getArchivedQuizzes)
 	e.GET("/profil", qph.getQuizProfile)
+
+	e.GET("/username", qph.usernamePage)
```

Comment:
Bruk norsk URL siden disse er for norske  brukere

Reply:
Gjort